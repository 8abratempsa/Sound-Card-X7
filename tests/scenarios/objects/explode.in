octosql "SELECT field1, field3->* FROM fixtures/test.json" -o json
