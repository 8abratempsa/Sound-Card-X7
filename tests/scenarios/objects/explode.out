{"field1":"value","field4":"eulav","field5":24,"field6":null}
{"field1":"value","field4":null,"field5":"eulav","field6":"value"}
